data/src/main/java/com/teamdontbe/data/repositoryimpl/MyPageRepositoryImpl.kt

```diff
@@ -0,0 +1,50 @@
+package com.teamdontbe.data.repositoryimpl
+
+import com.teamdontbe.data.datasource.MyPageDataSource
+import com.teamdontbe.domain.entity.FeedEntity
+import com.teamdontbe.domain.entity.MyPageCommentEntity
+import com.teamdontbe.domain.entity.MyPageUserProfileEntity
+import com.teamdontbe.domain.repository.MyPageRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import javax.inject.Inject
+
+class MyPageRepositoryImpl
+@Inject constructor(
+    private val myPageDataSource: MyPageDataSource,
+) : MyPageRepository {
+    override suspend fun getMyPageUserProfile(viewMemberId: Int): Flow<MyPageUserProfileEntity?> {
+        return flow {
+            val result = kotlin.runCatching {
+                myPageDataSource.getMyPageUserProfileSource(viewMemberId).data?.toMyPageUserProfileEntity()
+            }
+            emit(result.getOrDefault(MyPageUserProfileEntity(0, "", "", "", 0)))
```

Comment:
```suggestion
            emit(result.getOrDefault(MyPageUserProfileEntity(-1, "", "", "", -1)))
```

p2 : 0은 리스트 포지션에서 받아올 수도 있어서, -1로 안전하게 디폴크값 줘보는건 어떤가유

Reply:
[2bba1be](https://github.com/TeamDon-tBe/ANDROID/pull/103/commits/2bba1bed92c1b442df3a7a158ebafd367c3abb19)